src/appleseed.studio/utility/miscellaneous.h

```diff
@@ -82,6 +82,9 @@ QString combine_name_and_shortcut(const QString& name, const QKeySequence& short
 // Check whether a file exists.
 bool file_exists(const QString& path);
 
+// Load a glsl shader from file into a string
+const std::string load_gl_shader(const QString& resource_path);
```

Comment:
Remove const on return value. The mix between `QString` and `std::string` on this function is... curious. Can we stick to `QString`?